src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -299,6 +299,17 @@ pub async fn process(request: &Transaction<'_>) -> Result<Response, Error> {
     }
     super::keypath::warn_unusual_keypath(&params, params.name, request.keypath()).await?;
 
+    // Show chain confirmation only for known non-mainnet networks
+    if super::params::is_known_non_mainnet(params.chain_id) {
+        confirm::confirm(&confirm::Params {
+            title: "Confirm Chain",
+            body: &format!("Sign transaction on\n{}", params.name),
+            accept_is_nextarrow: true,
+            ..Default::default()
+        })
+        .await?;
```

Comment:
In BTC, we have this wording currently:


Title: `Spend from`
Body: `<account name>`

For consistency you could do the same, but simply use `body: params.name.into()`?

Reply:
hm, `Spend from\n Optimism` doesn't sound quite right, imo you spend `from` your account, but `on` a network
Also since it's not BTC, you might not actually be spending anything, you could just be signing a tx, like `Approve` to approve a token spend, or bidding on something and so on

Reply:
:+1: 

I'd drop the "Confirm chain" as it does not add much info and sounds a bit too technical for users that only use mainnet in the app.